course_flow/models.py

```diff
@@ -104,6 +122,8 @@ class Node(models.Model):
     )
     classification = models.PositiveIntegerField(choices=NODE_TYPES, default=1)
 
+    column = models.ForeignKey("Column", on_delete=models.PROTECT,null=True)
+
```

Comment:
Why the `on_delete=models.PROTECT`? You might be looking for `on_delete=models.SET_NULL`

Reply:
Really good catch.

Reply:
This was actually done on purpose. Once a node has been assigned to a column, we shouldn't be able to set it back to null. Later I'll add something that handles deleting columns better (re-assigning nodes to another column automatically) but until then I decided one should not be able to delete a column until all nodes are removed from it.